STARTERKIT/js/script.js

```diff
@@ -1,31 +0,0 @@
-/**
```

Comment:
it suits us better because init.js makes more sense as its the initialization file